neovim/ui/gtk_ui.py

```diff
@@ -152,7 +152,11 @@ def _nvim_resize(self, columns, rows):
         self._cell_pixel_width = cell_pixel_width
         self._cell_pixel_height = cell_pixel_height
         self._screen = Screen(columns, rows)
-        self._window.resize(pixel_width, pixel_height)
+        prev_rect = self._drawing_area.get_allocation()
```

Comment:
It was some time ago I coded gtk, but to me this looks a bit _ad-hoc_. Won't we just adjust the size of the drawing area and let gtk layouting handle the rest? (especially if we intend the widget to embedded in more complex layouts)